### ISSUE TITLE
OpenSSH mapping in Windows.

### ISSUE THREAD

**Issue by mdecrevoisier** [SigmaHQ/sigma#3473]
Hi,
I was looking for the product and service mapping that I should use in regards of the Windows Event Channel **'OpenSSH/Operational'**, but I couldn't find anything except in the Linux part. Could you clarify which mapping we should use for this channel ? Please not that this is not related to the WSL topic. This is a native Channel that can be found in Windows 10.

Maybe a suggestion:
-   product: windows
-   service: sshd

SIGMA log sources: https://github.com/SigmaHQ/sigma/wiki/Log-Sources
My Windows OpenSSH rule: https://github.com/mdecrevoisier/SIGMA-detection-rules/blob/main/windows-openssh/win-os-OpenSSH%20server%20listening.yaml

Many thanks
Michel de Crevoisier

![ssh windows](https://user-images.githubusercontent.com/52909656/188973055-fe609296-2438-4432-bdfc-98a37c89c861.png)

**Comment by frack113** [comment]
If we keep the logic for windows channel
```
product: windows
service: openssh
```
sshd is the process name 
On linux `service: sshd` is for the log of the ssh deamon.

**Comment by Neo23x0** [comment]
Yeah, but we would also need new mappings in every backend config. 

e.g., this PR contains changes that are needed in the backends: 
https://github.com/SigmaHQ/sigma/pull/1590/files
